### ISSUE TITLE
INOP label for WX Radar removed, but WX Radar is not working.

### ISSUE THREAD

**Issue by GeheimagentNr1** [flybywiresim/a32nx#8042]
### Aircraft Version

Experimental

### Build info

```json
{
    "built": "2023-05-23T08:10:56+00:00",
    "ref": "refs/heads/experimental",
    "sha": "7ac70bfd908988bfdc68879b16a169d26bf0c72c",
    "actor": "Saschl",
    "event_name": "manual",
    "pretty_release_name": "experimental:7ac70bfd",
    "version": "v0.10.0-exp.7ac70bf"
}
```

### Describe the bug

The INOP label was removed for Terrain on ND and WX Radar a while ago, but only Terrain on ND is working and WX Radar not.

### Expected behavior

Separate INOP labels for Terrain on ND and WX Radar.
For Terrain on ND a "Removed INOP label" and a "normal INOP label" for the WX Radar.

### Steps to reproduce

Check the INOP labels on the WX Radar and the Terrain on ND.

**Comment by alokm95** [comment]
WX is not supported as Asobo havent implemented API yet

**Comment by tracernz** [comment]
Yeah, limitation of how the labels are done in the model. We will address this in future.
